publish-maven.gradle

```diff
@@ -0,0 +1,49 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.artifactory'
+
+publishing {
```

Comment:
We don't want this one as we already are doing this via https://github.com/spring-projects-experimental/spring-pulsar/blob/main/gradle/publish-artifactory.gradle